### PR TITLE
WIP - MTV-1358: Release Notes for MTV 2.6.5

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [MTV-1358](https://issues.redhat.com/browse/MTV-1358)

### PREVIEW

* [Resolved issues 2.6.5](https://anarnold97.github.io/MTA-Preview/MTV/rn-2-6-5.html#resolved-issues-26-5_release-notes)

* [Updated known issues](https://anarnold97.github.io/MTA-Preview/MTV/rn-2-6-5.html#known-issues-26_release-notes)

### CODE REVIEW

File: documentation/modules/rn-2.6.adoc [hunk 1]
```diff
@@ -141,6 +141,13 @@ The validation service includes default validation rules for virtual machines fr
 
 This release has the following resolved issues:
 
+[id="resolved-issues-26-5_{context}"]
+=== Resolved issues 2.6.5
+
+.VMware RHEL 8 network interface name changes during the migration
+
+In earlier releases of {project-short}, during the migration of a simple Red Hat Enterprise Linux (RHEL8) virtual machine (VM) from VMware 7 to {ocp} (OCP) version 4.13, the name of the network interfaces is modified, and the static IP configuration for the VM is no longer functional. This issue has been resolved for static IPs in {project-short} 2.6.5. link:https://issues.redhat.com/browse/MTV-595[(MTV-595)]
```
Comment (fabiand) [inline documentation/modules/rn-2.6.adoc:149]: From
`during the migration of a simple Red Hat Enterprise Linux (RHEL8) virtual machine (VM) from VMware 7 to {ocp} (OCP) version 4.13`

to
`during the migration of Rocky 8, CentOS 7.2+, and Ubuntu 22 virtual machines (VM) from VMware to {ocp}`

File: documentation/modules/rn-2.6.adoc [hunk 2]
```diff
@@ -309,6 +316,10 @@ For a complete list of all resolved issues in this release, see the list of link
 
 This release has the following known issues:
 
+.Migrating RHEL 7.0 and 7.1 from VMware 7 to {ocp} 4.13, the name of the network interfaces changes
```
Comment (fabiand) [inline documentation/modules/rn-2.6.adoc:319]: `.Migrating RHEL 7.0 and 7.1 from VMware 7 to {ocp} 4.13, the name of the network interfaces changes`

to
`.Migrating RHEL and CentOS 7.0 and 7.1 from VMware to {ocp}, the name of the network interfaces changes`

File: documentation/modules/rn-2.6.adoc [hunk 3]
```diff
@@ -141,6 +141,13 @@ The validation service includes default validation rules for virtual machines fr
 
 This release has the following resolved issues:
 
+[id="resolved-issues-26-5_{context}"]
+=== Resolved issues 2.6.5
+
+.VMware RHEL 8 network interface name changes during the migration
```
Comment (anarnold97) [inline documentation/modules/rn-2.6.adoc:147]:
```suggestion
.VMware Linux interface name changes during migration
```

File: documentation/modules/rn-2.6.adoc (hunk 2 above)
Comment (anarnold97) [inline documentation/modules/rn-2.6.adoc:319]:
```suggestion
.Migrating RHEL 7.0 and 7.1 from VMware 7 to {ocp} 4.13, the name of the network interfaces changes
```

File: documentation/modules/rn-2.6.adoc [hunk 4]
```diff
@@ -141,6 +141,13 @@ The validation service includes default validation rules for virtual machines fr
 
 This release has the following resolved issues:
 
+[id="resolved-issues-26-5_{context}"]
+=== Resolved issues 2.6.5
+
+.VMware Linux interface name changes during migration
+
+In earlier releases of {project-short}, during the migration of a simple Red Hat Enterprise Linux (RHEL8) virtual machine (VM) from VMware 7 to {ocp} (OCP) version 4.13, the name of the network interfaces is modified, and the static IP configuration for the VM is no longer functional. This issue has been resolved for static IPs in {project-short} 2.6.5. link:https://issues.redhat.com/browse/MTV-595[(MTV-595)]
```
Comment (anarnold97) [inline documentation/modules/rn-2.6.adoc:149]:
```suggestion
In earlier releases of {project-short}, 

during the migration of Rocky 8, CentOS 7.2+, and Ubuntu 22 virtual machines (VM) from VMware to {ocp} 
(OCP) version 4.13, the name of the network interfaces is modified, and the static IP configuration for the VM is no longer functional. This issue has been resolved for static IPs in {project-short} 2.6.5. link:https://issues.redhat.com/browse/MTV-595[(MTV-595)]
```

File: documentation/modules/rn-2.6.adoc [hunk 5]
```diff
@@ -141,6 +141,16 @@ The validation service includes default validation rules for virtual machines fr
...
 
+[id="resolved-issues-26-5_{context}"]
+=== Resolved issues 2.6.5
+
+.VMware Linux interface name changes during migration
+
+In earlier releases of {project-short}, 
+
+during the migration of Rocky 8, CentOS 7.2+, and Ubuntu 22 virtual machines (VM) from VMware to {ocp} 
+(OCP) version 4.13, the name of the network interfaces is modified, and the static IP configuration for the VM is no longer functional. This issue has been resolved for static IPs in {project-short} 2.6.5. link:https://issues.redhat.com/browse/MTV-595[(MTV-595)]
```
Comment (anarnold97) [inline documentation/modules/rn-2.6.adoc:152]:
```suggestion
In earlier releases of {project-short}, during the migration of Rocky 8, CentOS 7.2+, and Ubuntu 22 virtual machines (VM) from VMware to {ocp} (OCP) version 4.13, the name of the network interfaces is modified, and the static IP configuration for the VM is no longer functional. This issue has been resolved for static IPs in {project-short} 2.6.5. link:https://issues.redhat.com/browse/MTV-595[(MTV-595)]
```

File: documentation/modules/rn-2.6.adoc [hunk 6]
```diff
@@ -141,6 +141,13 @@ The validation service includes default validation rules for virtual machines fr
 
 This release has the following resolved issues:
 
+[id="resolved-issues-26-5_{context}"]
+=== Resolved issues 2.6.5
+
+.VMware Linux interface name changes during migration
+
+In earlier releases of {project-short}, during the migration of Rocky 8, CentOS 7.2+, and Ubuntu 22 virtual machines (VM) from VMware to {ocp} (OCP) version 4.13, the name of the network interfaces is modified, and the static IP configuration for the VM is no longer functional. This issue has been resolved for static IPs in {project-short} 2.6.5. link:https://issues.redhat.com/browse/MTV-595[(MTV-595)]
```
Comment (fabiand) [inline documentation/modules/rn-2.6.adoc:149]: Please drop ` version 4.13` - as all OCP versions are affected.

Let's be specific about the verified Osses:

`resolved for static IPs in Rocky 8, Centos 7.2+, Ubuntu 22`

File: documentation/modules/rn-2.6.adoc [hunk 7]
```diff
@@ -309,6 +316,10 @@ For a complete list of all resolved issues in this release, see the list of link
 
 This release has the following known issues:
 
+.Migrating RHEL and CentOS 7.0 and 7.1 from VMware to {ocp}, the name of the network interfaces changes
+
+When migrating a simple Red Hat Enterprise Linux (RHEL) 7.0 and 7.1, as well as Rocky Linux and CentOS 7.0 and 7.1, virtual machine (VM) from VMware 7 to {ocp} 4.13, the name of the network interfaces changes and the static IP configuration for the VM no longer functions. This issue is caused by RHEL 7.0 and 7.1 still requiring `virtio-transitional`. Workaround: Manually update the guest to RHEL 7.2 or update the VM specification post-migration to use transitional. link:https://issues.redhat.com/browse/MTV-1382[(MTV-1382)]
```
Comment (fabiand) [inline documentation/modules/rn-2.6.adoc:321]: Please change
`When migrating a simple Red Hat Enterprise Linux (RHEL) 7.0 and 7.1, as well as Rocky Linux and CentOS 7.0 and 7.1, virtual machine (VM) from VMware 7 to {ocp} 4.13,`

to

`When migrating older Linux distributions, such as CentOS 7.0 and 7.1 virtual machines (VM) from VMware to {ocp},`

**Review by fabiand — COMMENTED** [review]
Two changes, then we should be good

File: documentation/modules/rn-2.6.adoc (hunk 7 above)
Comment (anarnold97) [inline documentation/modules/rn-2.6.adoc:321]:
```suggestion
When migrating older Linux distributions, such as CentOS 7.0 and 7.1 virtual machines (VM) from VMware to {ocp}, the name of the network interfaces changes and the static IP configuration for the VM no longer functions. This issue is caused by RHEL 7.0 and 7.1 still requiring `virtio-transitional`. Workaround: Manually update the guest to RHEL 7.2 or update the VM specification post-migration to use transitional. link:https://issues.redhat.com/browse/MTV-1382[(MTV-1382)]
```

File: documentation/modules/rn-2.6.adoc [hunk 8]
```diff
@@ -309,6 +316,10 @@ For a complete list of all resolved issues in this release, see the list of link
 
 This release has the following known issues:
 
+.Migrating RHEL and CentOS 7.0 and 7.1 from VMware to {ocp}, the name of the network interfaces changes
```
Comment (anarnold97) [inline documentation/modules/rn-2.6.adoc:319]:
```suggestion
.Migrating older Linux distributions from VMware to {ocp}, the name of the network interfaces changes
```

**Review by fabiand — COMMENTED** [review]
lgtm! Thanks

**Pull request merged**